apps/io/search_ingest.py

```diff
@@ -106,3 +110,89 @@ def _get_query(self, req):
         args = getattr(req, "args", {})
         query = json.loads(args.get("source")) if args.get("source") else {"query": {"filtered": {}}}
         return query
+
+
+class SearchIngestServiceAsync(AsyncBaseService):
```

Comment:
Is there a reason why you've kept the original `SearchIngestService` and added this one, instead of upgrading that service to async?

Reply:
yes, I want to keep the old code working until I do the full switch, but I want to test with few async search providers first.